### PR TITLE
improved responsiveness

### LINKED ISSUE #470: Enhance responsiveness
## **Describe the bug**
Make a better responsive webpage

## **Possible solution**
I would like to make responsive this sharpen skill section

## **Screenshots**
Screenshot (571)

I am a GSSOC'21 participant and would like to work on this issue.
please assign this issue to me if you find it valid.

### PR DESCRIPTION
**Pull request by Saumya-07**
# Description

Summary of the change and which issue is fixed.
-I have improved responsiveness in the sharpen-skill section.
-I changed the table to grid and added styling as required and took care of styling as before.

Fixed  #470 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## UI /UX changes
   Attach gif or screenshot for changes.
- Before :

https://user-images.githubusercontent.com/68998355/118401301-d2964800-b682-11eb-9e51-5a634747e570.mp4

- After :
  
https://user-images.githubusercontent.com/68998355/118402946-ceb9f400-b689-11eb-9fbc-11d11796649f.mp4

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -198,23 +198,19 @@ <h1 class="text-center">About Us</h1>
...
-        <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=eTXd89t8ngI&list=PLd3UqWTnYXOmx_J1774ukG_rvrpyWczm0" target="_blank">Java</a></th>
-      </tr>
-      <tr>
-        <th><i class="far fa-gem"></i>&nbsp; <a href="https://www.youtube.com/watch?v=t_ispmWmdjY" target="_blank">Ruby</a></th>
-        <th><i class="fab fa-gofore"></i>&nbsp; <a href="https://www.youtube.com/watch?v=YS4e4q9oBaU" target="_blank">Golang</a></th>
-        <th><i class="fab fa-swift"></i>&nbsp; <a href="https://www.youtube.com/watch?v=Ulp1Kimblg0"  target="_blank">Swift</a></th>
-      </tr>
-      </table>
+    <div class="table">
+      <div class='tr'>
```
Comment (harikesh409) [inline index.html:202]: change the class name to something more meaningful.

File: index.html [hunk 2]
```diff
@@ -198,23 +198,19 @@ <h1 class="text-center">About Us</h1>
...
-        <th><i class="devicon-cplusplus-line-wordmark fa-lg"></i>&nbsp; <a href="https://www.cplusplus.com/" target="_blank">C++</a></th>
-        <th><i class="fab fa-java fa-lg"></i>&nbsp; <a href="https://www.youtube.com/watch?v=eTXd89t8ngI&list=PLd3UqWTnYXOmx_J1774ukG_rvrpyWczm0" target="_blank">Java</a></th>
-      </tr>
-      <tr>
-        <th><i class="far fa-gem"></i>&nbsp; <a href="https://www.youtube.com/watch?v=t_ispmWmdjY" target="_blank">Ruby</a></th>
-        <th><i class="fab fa-gofore"></i>&nbsp; <a href="https://www.youtube.com/watch?v=YS4e4q9oBaU" target="_blank">Golang</a></th>
-        <th><i class="fab fa-swift"></i>&nbsp; <a href="https://www.youtube.com/watch?v=Ulp1Kimblg0"  target="_blank">Swift</a></th>
-      </tr>
-      </table>
+    <div class="table">
```
Comment (harikesh409) [inline index.html:201]: change the class name to something more meaningful.

File: index.html [hunk 3]
```diff
@@ -198,23 +198,19 @@ <h1 class="text-center">About Us</h1>
...
+      <div class='tr'>
+        <div class='td'><i class="fas fa-atom"></i>&nbsp; <a href="https://www.geeksforgeeks.org/data-structures/" target="_blank">Data Structures</a></div>
```
Comment (harikesh409) [inline index.html:203]: change the class name to something more meaningful.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Change the class names `tr`, `td` and `table` to something more meaningful.

**Comment by Saumya-07** [comment]
Added the required changes as mentioned @harikesh409

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**